### PR TITLE
Fix include instructions for twilio-video.js

### PR DESCRIPTION
**Pull request by markandrus**
@aaalaniz

### CODE REVIEW

File: video/getting-started/create-video-client/create-video-client.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 // If you are requiring twilio-video from CommonJS,
```
Comment (manjeshbhargav) [inline video/getting-started/create-video-client/create-video-client.js]: @markandrus I would rewrite this as:
```javascript
// If you are using CommonJS module pattern, then you can include the SDK like this:
// const Video = require('twilio-video');
```

File: video/getting-started/setup-local-media/setup-local-media.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
 // If you are requiring twilio-video from CommonJS,
```
Comment (manjeshbhargav) [inline video/getting-started/setup-local-media/setup-local-media.js]: @markandrus Same as above.

**Comment by manjeshbhargav** [comment]
@markandrus Looks good. I'll leave it up to you to decide whether you want to include my suggestion about wording the comment for `require()` differently.

File: video/getting-started/create-video-client/create-video-client.js (hunk 1 above)
Comment (markandrus) [inline video/getting-started/create-video-client/create-video-client.js]: I think "requiring" is a little more direct than "use", and makes sense in the context of CommonJS.

**Comment by markandrus** [comment]
Thanks for the feedback @manjeshbhargav

**Pull request merged**